src/main/java/api/Management.java

```diff
@@ -35,9 +36,23 @@ public Response deleteDeployKey(String id) throws Exception {
   }
 
   @POST
-  @Path("/deploykey/{id}")
-  public Response createDeployKey(String id) throws Exception {
-    DeployKey deployKey = deployKeyService.createDeployKey(id);
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  @Path("/deploykey")
+  public Response createDeployKey(  @RestForm String resourceType,
+  @RestForm String scope,
```

Comment:
To get request validation we should consider using Enum like in [the frontend part you did use constants](https://github.com/PacoVK/tapir/pull/445/files#diff-e6a0f00a27ed8fbbb20792436ea353b818a46cd6c4a9d78391c11bc266df8517R17) otherwise scope parameter can be invalid but is required to match one of the given values

Reply:
I've implemented it, but I'm no Java dev, so it might not be the best way to do it